src/styles/index.scss

```diff
@@ -1,3 +1,237 @@
+$--blue-color: #00acdc;
+$nav-links-color: #060b35;
+$card-width: 200px;
+$font-family: 'Roboto', sans-serif;
+$main-color: #060b35;
+$secondary-color: #616070;
+$button-bg-color: #00acdc;
+$button-bg-color-hover: #fff;
+$button-text-color: white;
+
 body {
   margin: 0;
+  font-family: $font-family;
+  color: $main-color;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  &__logo {
+    width: 40px;
+    height: 40px;
+    display: block;
+  }
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
The font-family property is already defined in the body selector. There's no need to redefine it in other selectors unless you want to change the font. If 'Roboto' is the main font of your project, you can remove this line.